frame/system/src/lib.rs

```diff
@@ -296,7 +297,7 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
-		/// A dispatch that will fill the block weight up to the given ratio.
+		/// A dispatch that will fill the block weight up to the given ratio.a
```

Comment:
probably introduce in merge
```suggestion
		/// A dispatch that will fill the block weight up to the given ratio.
```